lib/oneview-sdk/exceptions.rb

```diff
@@ -11,45 +11,65 @@
 
 # Contains all the custom Exception classes
 module OneviewSDK
-  class ConnectionError < StandardError # Cannot connect to client/resource
+  # Error class allowing storage of a data attribute
+  class OVError < ::StandardError
+    attr_accessor :data
+
+    def initialize(msg = nil, data = nil)
+      @data = data
+      super(msg)
+    end
+
+    # Shorthand method to raise an error.
+    # @example
+    #   OneviewSDK::OVError.raise! 'Message', { data: 'stuff' }
+    def self.raise!(msg = nil, data = nil)
+      raise new(msg, data)
+    end
+  end
+
+  class ConnectionError < OVError # Cannot connect to client/resource
```

Comment:
Why we don't have a default message for each of the errors? These comments on each error declaration are good candidates for them.